tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1045,22 +1049,26 @@ if (/* Comments */ true /* Comments */) {
 } else {}
 
 
-  if (2 /* MB_OVERLOAD_STRING */ & (int) ini_get('mbstring.func_overload')) {
-      mb_internal_encoding('ASCII');
-  }~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+if (2 /* MB_OVERLOAD_STRING */ & (int) ini_get('mbstring.func_overload')) {
+  mb_internal_encoding('ASCII');
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 <?php
 // Comment
 if (1) {
     // Comment
     // Comment
     echo "Foo"; // Comment
     // Comment
-} elseif (2) {
+}
+// Comment
+elseif (2) {
```

Comment:
this does not look like valid PSR-2: https://www.php-fig.org/psr/psr-2/#51-if-elseif-else

but to make it valid we would have to move (or remove) the comment

Reply:
Oh right, I didn’t even think of that! Shall we just move the comment to the top of the body of the elseif/else?

Reply:
I agree looks very ugly, but PSR-12 don't have section about comments and where they should be placed. Let's do it pretty and move this comments in some other place.

Reply:
@evilebottnawi I agree - what do you think about how it's done now? (see below)